.cirrus.yml

```diff
@@ -465,8 +465,9 @@ osx_alt_build_task:
     # This host is/was shared with potentially many other CI tasks.
     # The previous task may have been canceled or aborted.
     prep_script: &mac_cleanup "contrib/cirrus/mac_cleanup.sh"
+    clone_script: *full_clone
```

Comment:
It's really hard to tell where this is going wrong from the output :sad:  But I will point out that it was never intended to run on anything other than Linux.  It's possible bash and/or git on the mac is "too old".  Maybe try adding a `set -x` (temporarily) to the start of that anchor (i.e. the script in the `build` job).  Otherwise I fear there may be no shorter way than ssh'ing into the machine and seeing what's actually in `$CIRRUS_WORKING_DIR`.

Reply:
```bash
$ automation/mac_pw_pool/InstanceSSH.sh MacM1-2
+ ssh -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no -o CheckHostIP=no -F /dev/null -o LogLevel=ERROR -o ConnectTimeout=13 ec2-user@ec2-44-205-13-254.compute-1.amazonaws.com

    ┌───┬──┐   __|  __|_  )
    │ ╷╭╯╷ │   _|  (     /
    │  └╮  │  ___|\___|___|
    │ ╰─┼╯ │  Amazon EC2
    └───┴──┘  macOS Sonoma 14.1


The default interactive shell is now zsh.
To update your account to use zsh, please run `chsh -s /bin/zsh`.
For more details, please visit https://support.apple.com/kb/HT208050.
MacM1-2:~ ec2-user$ sudo su - MacM1-2-worker
MacM1-2-worker@MacM1-2 ~ % cd ci/task-5534374309920768
MacM1-2-worker@MacM1-2 task-5534374309920768 % git status
fatal: not a git repository (or any parent up to mount point /Users)
Stopping at filesystem boundary (GIT_DISCOVERY_ACROSS_FILESYSTEM not set).
MacM1-2-worker@MacM1-2 task-5534374309920768 %
```